src/services/formatting/smartIndenter.ts

```diff
@@ -531,9 +531,23 @@ namespace ts.formatting {
             return false;
         }
 
-        export function nodeWillIndentChild(parent: TextRangeWithKind, child: TextRangeWithKind | undefined, indentByDefault: boolean): boolean {
+        export function nodeWillIndentChild(parent: TextRangeWithKind, child: Node | undefined, indentByDefault: boolean): boolean {
             const childKind = child ? child.kind : SyntaxKind.Unknown;
+
             switch (parent.kind) {
+                case SyntaxKind.VariableDeclaration:
+                case SyntaxKind.PropertyAssignment:
+                case SyntaxKind.ObjectLiteralExpression:
+                    if (childKind === SyntaxKind.ObjectLiteralExpression) {
+                        const sourceFile = child.getSourceFile();
+                        if (sourceFile) {
+                            // May not be defined for synthesized nodes.

```

Comment:
This was meant to describe the `if (sourceFile)`? I think it should go on the same line then.